src/amo/components/AddonReview/styles.scss

```diff
@@ -35,6 +35,8 @@
 
 .AddonReview-textarea,
 .AddonReview-submit {
+  @include font-regular();
```

Comment:
Is this font messed up for all textareas? If so, we should be changing this at the tag-level, not just here.

Reply:
The inputs throughout the site just have 'sans serif', example in search bar.

Reply:
I realized, we have input only at two places on complete website, one is submit review and other is search box, so shall we use `Fira sans`everywhere?